# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/css-grid/grid-fragmentation-015-ref.html
# Commit: 0186165ee1d8
# Full Hash: 0186165ee1d84f56362ba16bbda4eefa1d47fd60
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 05:19:55
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r 9cf1851c76dc -r 0186165ee1d8 layout/reftests/css-grid/grid-fragmentation-015-ref.html
--- a/layout/reftests/css-grid/grid-fragmentation-015-ref.html	Mon Nov 25 15:47:38 2019 +0000
+++ b/layout/reftests/css-grid/grid-fragmentation-015-ref.html	Mon Nov 25 18:05:16 2019 +0000
@@ -72,7 +72,7 @@
 <!-- grid wrapped in FIELDSET block -->
 <div class="columns" style="height: 40px; margin-top:70px">
 <div style="padding-top:2px; background:grey">
-<div style="display:block; overflow:hidden; border:none; padding:0; margin:0">
+<div style="border:none; padding:0; margin:0">
 <div class="grid">
 <span style="grid-row:span 2"><x></x></span>
 <span><x></x></span>